Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp

```diff
@@ -182,7 +182,7 @@ struct CombinatorialKalmanFilterOptions {
   std::reference_wrapper<const CalibrationContext> calibrationContext;
 
   /// The source link accessor
-  SourceLinkAccessor sourcelinkAccessor;
+  SourceLinkAccessor sourceLinkAccessor;
```

Comment:
This is a breaking change. Can we reflect that in the PR title?

Reply:
Thanks for the patch.